packages/rainbowkit/src/wallets/walletConnectors/omni/omni.ts

```diff
@@ -4,19 +4,19 @@ import { isAndroid } from '../../../utils/isMobile';
 import { Wallet } from '../../Wallet';
 import { getWalletConnectConnector } from '../../getWalletConnectConnector';
 
-export interface SteakOptions {
+export interface OmniOptions {
   chains: Chain[];
 }
 
-export const steak = ({ chains }: SteakOptions): Wallet => ({
-  id: 'steak',
-  name: 'Steakwallet',
-  iconUrl: async () => (await import('./steak.svg')).default,
+export const omni = ({ chains }: OmniOptions): Wallet => ({
+  id: 'omni',
+  name: 'Omni',
+  iconUrl: async () => (await import('./omni.svg')).default,
   iconBackground: '#000',
   downloadUrls: {
     android: 'https://play.google.com/store/apps/details?id=fi.steakwallet.app',
     ios: 'https://apps.apple.com/np/app/steakwallet/id1569375204',
```

Comment:
should the `android` and `ios` download URLs also be updated? or is the rebranded app not published yet?

Reply:
Another good catch, iOS yes but Android no. We didn't create a new app, so we'll always maintain the steakwallet bundle ID